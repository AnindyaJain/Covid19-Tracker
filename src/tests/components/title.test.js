import React from 'react';
// import { shallow } from 'enzyme';
import { render } from '@testing-library/react';
import  Title  from '../../components/title';

describe('<Title />', ()=> {
    it('renders the <Title /> with populated data', ()=>{
        const {container, getByText } = render(
            <Title>
                <Title.Heading>Covid19Tracker</Title.Heading>
            </Title>
        );
        expect(getByText('Covid19Tracker')).toBeTruthy();
        expect(container.firstChild).toMatchSnapshot();
    });
});
