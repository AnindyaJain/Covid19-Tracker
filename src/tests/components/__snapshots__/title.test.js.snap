// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> renders the <Title /> with populated data 1`] = `
<div
  class="title__Container-sc-19e1hlj-0 iPLRZX"
>
  <div
    class="title__Heading-sc-19e1hlj-1 kgLoiP"
  >
    Covid19Tracker
  </div>
</div>
`;
